docs/tutorial.md

```diff
@@ -146,11 +167,11 @@ When the software template runs, you should see that Backstage successfully fetc
 
 ![Tutorial scaffolder page 3](images/tutorial-scaffolder-3.png "Tutorial scaffolder page 3")
 
-Go to the new component's page: `http://<your backstage app>/catalog/default/component/my-tutorial-service`.  You should see an AWS Proton Service card in the component's Overview tab.
+Go to the new component's page: `http://<your backstage app>/catalog/default/component/my-tutorial-service`.  You should see an **AWS Proton Service** card in the component's **Overview** tab. The service has two service instances, one in each of the two environments, **dev** and **prod**.
```

Comment:
"two environments" - since the tutorial has one env, maybe should be something like "The service has two service instances deployed to the environment, one for dev and one for prod"?